examples/databricks/example_databricks_workflow.py

```diff
@@ -53,19 +53,23 @@
         databricks_conn_id=DATABRICKS_CONN_ID,
         job_clusters=job_cluster_spec,
         notebook_params=[],
+        notebook_packages=[
+            {"pypi": {"package": "simplejson"}},
+        ],
     )
     with task_group:
         notebook_1 = DatabricksNotebookOperator(
             task_id="notebook_1",
             databricks_conn_id=DATABRICKS_CONN_ID,
-            notebook_path="/Users/daniel@astronomer.io/Test workflow",
+            notebook_path="/Users/pankaj.koti@astronomer.io/Notebook1",
```

Comment:
Maybe we should find a shared folder for our notebooks so we don't keep changing this between PRs :). cc: @tatiana

Reply:
yes, using shared notebooks now https://github.com/astronomer/astronomer-cosmos/pull/159/commits/e7c47f8773d194039ec432eb6226801e0b2793d3